notebooks/06 Modelling - classification.ipynb

```diff
@@ -614,20 +614,31 @@
    "source": [
     "model_name = \"xgboost\"\n",
     "\n",
-    "# note neither weight nor sample_weights appears to affect training, so this is an unbalanced model\n",
+    "# note GridSearchCV does not currently pass in sample weights to XGBoost\n",
+    "# implement sklearn cross_validate instead\n",
     "# see https://discuss.xgboost.ai/t/multi-class-classification-weighting-for-unbalanced-datasets/2789\n",
-    "gsc = GridSearchCV(\n",
-    "    estimator=XGBClassifier(random_state=42, weight=weights),\n",
-    "    param_grid={},\n",
-    "    cv=5,\n",
+    "\n",
+    "clf = XGBClassifier(random_state=42)\n",
+    "cv_results = cross_validate(\n",
```

Comment:
May want to consider transitioning other models to this approach as currently GridsearchCV implemented with no parameter grid.

Reply:
Noted!